src/ruby/yast/rspec/matchers.rb

```diff
@@ -0,0 +1,40 @@
+require "rspec"
+require "yast"
+
+module Yast
+  module RSpec
+    # RSpec extension to add YaST-specific matchers
+    module Matchers
+      # Matches arguments of type YaST::Path whose string representation matches
+      # the provided regular expression
+      #
+      # @example
+      #   expect(Yast::SCR).to receive(:Read).with(path_matching(/^\.sysconfig\.nfs/))
+      def path_matching(match)
+        PathMatchingMatcher.new(match)
+      end
+
+      # @private
+      class PathMatchingMatcher
+        def initialize(expected)
+          @expected = Regexp.new(expected)
+        end
+
+        # RSpec 3 uses === while RSpec 2 uses ==
+        # Thus, implementing just == should work with both
```

Comment:
well, maybe add FIXME as it actually is not `==` and `===` is really proper method regarding behavior of class. but just nitpick